src/components/sbb-no-results/sbb-no-results.scss

```
@@ -0,0 +1,45 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-no-results-subtitle-color: var(--sbb-color-granite-default);
+  --sbb-no-results-image-margin-block: 0 var(--sbb-spacing-responsive-s);
+  --sbb-no-results-legend-margin-block: var(--sbb-spacing-responsive-xxxs) 0;
+  --sbb-no-results-action-margin-block: var(--sbb-spacing-responsive-xxxs) 0;
+}
+
+.sbb-no-results__container {
+  text-align: center;
+}
+
+::slotted([slot='title']) {
+  --sbb-title-margin-block-start: 0;
+  --sbb-title-margin-block-end: 0;
+
+  margin: 0;
+}
```

To fix the title margin:
```suggestion
.sbb-no-results__title {
  // Overwrite sbb-title default margin
  margin: 0;
}
```